tests/instrumentation-e2e-apps/apache-httpd/Dockerfile

```diff
@@ -0,0 +1,5 @@
+
+FROM httpd:2.4-alpine
+
+RUN sed -i "s#Listen 80#Listen 8080#g" /usr/local/apache2/conf/httpd.conf
+RUN chmod 777 -R /usr/local/apache2/
```

Comment:
I have an idea, is there a way to apply these changes at runtime (e.g. change the cmd command / use init container) ? 

The idea is to use just the upstream image.

Reply:
The problem is that, since we want to use a user different from than root, the user will not have permission to modify the folder permissions or the files.